scan/lib/scan/detect_values.rb

```diff
@@ -175,10 +175,26 @@ def self.detect_destination
 
       # building up the destination now
       if Scan.devices && Scan.devices.count > 0
```

Comment:
Does this seem like the correct fix?  https://github.com/fastlane/fastlane/pull/7807